src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,10 +34,40 @@ class AssetsInstallCommand extends ContainerAwareCommand
     const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
+    private $baseDir;
+    private $filesystem;
+
     /**
-     * @var Filesystem
+     * @param string          $baseDir
+     * @param Filesystem|null $filesystem
      */
-    private $filesystem;
+    public function __construct($baseDir = null, Filesystem $filesystem = null /**, $newApi = false*/)
```

Comment:
I think we already have such flags in the past, but we never introduced them commented.
Instead (if this trick is accepted by other reviewers), I'd suggest to uncomment and document this option in the docBlock, mentioning it'll only be necessary until 4.0, version in which it'll finally get removed.